release_bot/configuration.py

```diff
@@ -127,6 +130,14 @@ def load_release_conf(self, conf):
             msg = "Can't trigger on issue if 'github_username' is not known, disabling"
             self.logger.warning(msg)
             parsed_conf['trigger_on_issue'] = False
+        if parsed_conf.get('pypi_project') is None:
+            msg = "There is no different name for pypi_project"
+            self.logger.warning(msg)
+            parsed_conf['pypi_project'] = self.repository_name
+            setattr(self, self.pypi_project, self.repository_name)
```

Comment:
These two lines are way too complicated and not even doing what you want them to do, it should just:
```
self.pypi_project = repository_name
```
but since you also wrote the logic to parse repository name from setup.cfg, it could be a good idea to use it here